server/__tests__/reviewDataController.test.js

```diff
@@ -0,0 +1,99 @@
+const request = require('supertest');
+const app = require('../server');
+const mongoose = require('mongoose');
+const { noReviewData } = require('../constants/httpResponses');
+const { ReviewData } = require('../db/models/Review');
+
+let server;
+
+beforeAll(async () => {
+    await mongoose.connect(process.env.MONGO_URI_DEV, {
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+        useFindAndModify: false,
+    });
+    server = app.listen(5003, () => console.log('Test server started'));
+});
+
+afterAll(async () => {
+    await server.close();
+    await mongoose.disconnect();
+});
+
+describe('GET /api/reviews/info/:workspaceId', () => {
+    test('should get review data with properties avg and reviewCount', async () => {
+        const res = await request(app).get('/api/reviews/info/1');
+
+        expect(res.status).toBe(200);
+        expect(res.body).toHaveProperty('avg');
+        expect(res.body).toHaveProperty('reviewCount');
+    });
+
+    test('should return statusCode 404 with error message if no data is found', async () => {
+        const res = await request(app).get('/api/reviews/info/10000');
+
+        expect(res.status).toBe(404);
+        expect(res.body).toHaveProperty('message');
+        expect(res.body.message).toBe(noReviewData);
+    });
+});
+
+describe('POST /api/reviews/info', () => {
+    test('should create a new record for ReviewData', async () => {
+        const reviewDataBeforePost = await ReviewData.find();
+        const res = await request(app).post('/api/reviews/info');
+        const reviewDataAfterPost = await ReviewData.find();
+
+        expect(res.status).toBe(201);
+        expect(res.body).toHaveProperty('total');
+        expect(res.body).toHaveProperty('reviewCount');
+        expect(res.body).toHaveProperty('_id');
+        expect(res.body).toHaveProperty('workspaceId');
+        expect(res.body).toHaveProperty('workspaceSlug');
+        expect(res.body).toHaveProperty('reviews');
```

Comment:
If you are expecting back any particular values on the properties, you could use expect.objectContaining rather than reusing the toHaveProperty. That way you would only have to use one comparer. It would look something like this: 

<img width="476" alt="Screen Shot 2021-03-06 at 9 15 10 AM" src="https://user-images.githubusercontent.com/66852739/110218214-7a5df100-7e5c-11eb-9df0-d73ed935999b.png">

Reply:
At this point the seeding script generates randomized data, so I can't predict the values. But I will definitely ensure the data is consistent when I pick a database and use your suggestion instead.